source/upgrade.txt

```diff
@@ -141,17 +142,121 @@ Version 5.0 Breaking Changes
 
      ClusterSettings.builder()
          .hosts(Collections.singletonList(
-                 new ServerAddress("127.0.0.1", 27017)
+                new ServerAddress("127.0.0.1", 27017)
          ))
          .requiredReplicaSetName("replset")
          .build()
          .getMode()
```

Comment:
Is this the description you're looking for? 

You can see the existing content that's on master rn: https://github.com/mongodb/docs-java/blob/a541ef0f797d0ca44141e6efcee1c52c1822cc77/source/upgrade.txt#L124

It was moved to the breaking changes section and updated with an example, per [this comment](https://github.com/mongodb/docs-java/pull/507#discussion_r1462297527) in #507.

It looks like it appeared in #510 because of a messy rebase.

Reply:
1. The description of the `ConnectionId` changes id duplicated: at first it's on lines [66, 80], and then on lines [244, 258]. There is no such issue with https://github.com/mongodb/docs-java/pull/510, where only the second piece of the two aforementioned pieces is present.
2. The description about removal of `IterableCodec` and `MapCodec` does not contain the fix about `IterableCodecProvider` that was done in https://github.com/mongodb/docs-java/pull/510.

I haven't looked for more discrepancies. The above is enough to say that the rebase didn't go well. I am asking to please do a proper rebase before proceeding with requesting the review. In a properly rebased branch, I am expecting to see all the commits from https://github.com/mongodb/docs-java/pull/510, and then potentially more commits as needed on top of those. By "all the commits from " I mean commits with the same comments and diffs, unless rebasing a specific commit caused a non-trivial conflict and required a modification (in this case the commit's diff becomes different).

Reply:
I've re-fetched the original branch and re-rebased everything, being careful to keep things in order.

Reply:
Thank you 👍